### PR TITLE
Adds testing for macos

### PR DESCRIPTION
**Pull request by Trybnetic**
This PR will provide testing of pyndl on macos using travis

### CODE REVIEW

**Comment by derNarr** [comment]
This might help under Windows to get the netcdf4 headers:

http://geog.uoregon.edu/bartlein/courses/geog490/install_netCDF.html

**Comment by Trybnetic** [comment]
I would suggest merging this PR now and add windows testing later. Maybe windows CI is then also better supported by travis.

@derNarr what do you think?

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+git:
+  depth: 3
+  quiet: true
+
 python:
-    - 3.5
-    - 3.6
-    - 3.7
+  - 3.8
+
```
Comment (derNarr) [inline .travis.yml]: We should test against 3.5, 3.6, 3.7 as well, at least under linux. Is this still the case?

Comment (Trybnetic) [reply .travis.yml]: Thanks for noting that! Now it should be fixed!

**Comment by Trybnetic** [comment]
What version update should we do? As now macos is supported, we should at least do an update to v0.8.0. Maybe also to v1.0.0 depending how you prioritise the other issues.

**Comment by derNarr** [comment]
v0.7.1 should be the one, as this PR only fixes small (mostly maintenance things). The only "relevant" change is to `setup.py` which excludes openmp from Windows.

**Comment by Trybnetic** [comment]
Should we add `'Operating System :: MacOS',` in the `__classifiers__`?

**Comment by derNarr** [comment]
Yes.

**Pull request merged**